### PR TITLE
Remove BouncyCastle, JsonFx.Json, MarkerMetro and crypto DLLs.

### PR DESCRIPTION
**Pull request by crimsonred**
refactor(dependencies): Remove non required DLLs

Remove BouncyCastle, JsonFx.Json, MarkerMetro and crypto DLLs. Features works without the need of these libs.

### CODE REVIEW

File: PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeCommon.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@
 namespace PubNubAPI.Tests
 {
 
+#if !UNITY_WSA && !UNITY_WSA_10_0
```
Comment (budgetpreneur) [inline PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeCommon.cs:11]: Any reason why code is getting skipped for UNITY_WSA AND UNITY_WSA_10_0 ?

File: PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs [hunk 2]
```diff
@@ -1,12 +1,13 @@
-﻿using UnityEngine;
+﻿#if !UNITY_WSA && !UNITY_WSA_10_0
```
Comment (budgetpreneur) [inline PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs:1]: Any reason why code is getting skipped for UNITY_WSA AND UNITY_WSA_10_0 ?

File: PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeCommon.cs (hunk 1 above)
Comment (crimsonred) [reply PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeCommon.cs:11]: Please review the known limitations [here](https://docs.unity3d.com/Packages/com.unity.test-framework@1.0/manual/index.html)

File: PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs (hunk 2 above)
Comment (crimsonred) [reply PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs:1]: Please review the known limitations [here](https://docs.unity3d.com/Packages/com.unity.test-framework@1.0/manual/index.html)

Comment (budgetpreneur) [reply PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs:1]: @crimsonred  Were you able to run any example test on UWP to verify that it is working as expected.

Comment (crimsonred) [reply PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs:1]: yes, the example is running on UWP.

**Review by budgetpreneur — APPROVED** [review]
LGTM

**Comment by crimsonred** [comment]
@client-engineering-bot release

**Review by budgetpreneur — APPROVED** [review]
LGTM

**Review by techwritermat — APPROVED** [review]

**Pull request merged**